docs/pages/mainpage.md

```diff
@@ -2,13 +2,15 @@
 
 ## Introduction
 
-sACN is a C-language library that implements **ANSI E1.31: Lightweight streaming protocol for
-transport of DMX512 using ACN**, commonly referred to as **Streaming ACN** or **sACN**. The sACN
-library is designed to be portable and scalable to almost any sACN usage scenario, from lightweight
-embedded devices to large-scale data sending operations.
+*Streaming ACN (sACN)* is an ANSI standard for entertainment technology by
+[ESTA](http://tsp.esta.org) for transmission of DMX512 data over IP networks. sACN is widely used
+in the entertainment industry for real-time control of entertainment technology, especially
+lighting fixtures.
 
-**NOTE**: This open-source implementation is still early in development, and currently only
-receiving sACN is implemented.
+This repository contains a C-language library and a C++ wrapper library for communicating via sACN.
+
+**NOTE**: This open-source implementation is still in development. The merge receiver, source, and
```

Comment:
Assuming the implementations will be done before being merged to develop, this isn't necessary.